cp3pt0-deployment/setup_tenant.sh

```diff
@@ -158,6 +163,7 @@ function print_usage() {
     echo "   --excluded-namespaces string   Optional. Remove namespaces from this tenant, comma-delimited, e.g. 'ns1,ns2'"
     echo "   --license-accept               Required. Set this flag to accept the license agreement"
     echo "   --enable-private-catalog       Optional. Set this flag to use namespace scoped CatalogSource. Default is in openshift-marketplace namespace"
+    echo "   --with-minimal-rbac            Optional. File path to the minimal RBAC permissions required by the namespace scope operator"
```

Comment:
```suggestion
    echo "   --with-minimal-rbac  string           Optional. File path to the minimal RBAC permissions required by the namespace scope operator"
```

Hi @sgrube 
Could we add a `string` here to indicate there is a file path needed

Reply:
done